### PR TITLE
[libc][math] Add C23 math function fabsf128.

### PR DESCRIPTION
**Pull request by lntue**

### CODE REVIEW

**Comment by nickdesaulniers** [comment]
LGTM; but can you share the details of the GPU failure? cc @jhuber6

File: libc/config/linux/aarch64/entrypoints.txt [hunk 1]
```diff
@@ -354,6 +355,14 @@ set(TARGET_LIBM_ENTRYPOINTS
     libc.src.math.truncl
 )
 
+if(LIBC_COMPILER_HAS_FLOAT128)
+  list(APPEND TARGET_LIBM_ENTRYPOINTS
+    # math.h C23 _Float128 entrypoints
+    libc.src.math.copysignf128
```
Comment (nickdesaulniers) [inline libc/config/linux/aarch64/entrypoints.txt:361]: should copysign be a distinct commit?

Comment (lntue) [reply libc/config/linux/aarch64/entrypoints.txt]: Done.

**Review by michaelrj-google — APPROVED** [review]
LGTM

**Comment by jhuber6** [comment]
C23 math functions will be very interesting for the GPU considering that they have first-class support for `f16`, see https://godbolt.org/z/5aG3z15eb. Looking forward to it.

**Pull request merged**